dev/io.openliberty.jcache.internal_fat.infinispan/publish/servers/io.openliberty.jcache.internal.fat.dynamicupdate.1/server.xml

```diff
@@ -0,0 +1,49 @@
+<!--
+    Copyright (c) 2022 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server>
+
+	<featureManager>
+		<feature>appSecurity-3.0</feature>
+		<feature>jsp-2.3</feature>
+		<feature>servlet-4.0</feature>
+		<feature>timedexit-1.0</feature>
+		<feature>distributedSecurityCache-1.0</feature> <!-- TODO Remove when GA -->
```

Comment:
So... you might want to merge with release. I updated all the server.xml files so that common configuration was in include files. You will want to do the same here. The timedexit and dist... features have been moved into testingConfig.xml.

The httpEndpoint has been moved into httpEndpoint1.xml (and httpEndpoint2.xml if you use a 2nd server).

The applications have been moved into app*.xml (in this case I think appBasicAuth1.xml).

Reply:
This applies to the other server.xml files as well. I won't comment on them all.